packages/web-components/examples/all-charts.html

```diff
@@ -332,6 +332,175 @@ <h2>UN Map Comparisons</h2>
       variable="Count_Person"
       geoJsonProp="geoJsonCoordinatesUN"
     ></datacommons-map>
+    <h3>UN Regions</h3>
```

Comment:
nit: not sure we want to add all these maps to this example html since it's supposed to be used to show how different features of the web components are used .. what do you guys think, @dwnoble @juliawu?

Reply:
Let me move it to a different html?  I think its a very useful testing tool :)

Reply:
Any ideas where to put the new html?

Reply:
yeah moving to a different html sounds good to me & we could just keep it in the examples folder!

Reply:
Moving it to server/config/geojson/examples.html...

Reply:
Moved PTAL.

Reply:
lgtm!